front/app/containers/Admin/routes.tsx

```diff
@@ -100,9 +99,8 @@ const IndexElement = () => {
   });
 
   const { data: appConfiguration } = useAppConfiguration();
-  const { data: authUser } = useAuthUser();
 
-  if (isNilOrError(appConfiguration) || authUser === undefined) return null;
```

Comment:
authUser is `null` when not defined, contrary to what `useAuthUser's` types say.

Reply:
But the check was not needed here in any case, `usePermission` already checks `authUser` and `authUser` isn't used in this component.